csuntechlab/iSTART
# STR-271--Fixed review mode

# Description
Made review mode able to skip slides with no wait time, added review check for `set-module-progress` & `module-complete`.
  
# Testing
1. Check that in review mode, modules are not calling api's.
2. Check to see that you can skip slides instantly.
